qa/common/test_util.py

```diff
@@ -215,6 +215,27 @@ def validate_for_libtorch_model(input_dtype, output0_dtype, output1_dtype,
     return True
 
 
+def validate_for_python_model(input_dtype, output0_dtype, output1_dtype,
+                              input_shape, output0_shape, output1_shape):
+    """Return True if input and output dtypes are supported by a Python model."""
+
+    # STRING, FLOAT16 and UINT16 data types are not supported currently
```

Comment:
Why are these not supported?

Reply:
I thought it is a limitation with the HTTP api that cannot contain FP16 and UINT16. It seems to be working with the Python backend.